test/stage2/zir.zig

```diff
@@ -207,7 +207,7 @@ pub fn addCases(ctx: *TestContext) void {
         return;
     }
 
-    ctx.addZIRCompareOutput("hello world ZIR",
+    try ctx.addCompareOutput("hello world ZIR", .ZIR,
```

Comment:
I don't see these `try` as an improvement here. Note that this always runs as part of `zig build` (in debug mode) so in this case the `catch unreachable` that you would be avoiding here is equivalent to a panic. And OutOfMemory is the only possible error. In these files that have test cases, it should look extremely contributor friendly. We don't want any discouragement from adding test cases. Even something as trivial as the noise of `try`. If you really don`t want `catch unreachable`, fine, but still adding test cases should not fail, use `catch @panic("out of memory")` or something like that.

Reply:
Right, makes sense